src/components/Container/Container.tsx

```diff
@@ -37,15 +37,20 @@ export interface ContainerProps<T extends ElementType> {
   orientation?: Orientation;
   padding?: PaddingOptions;
   wrap?: WrapOptions;
+  height?: string;
+  maxHeight?: string;
+  minHeight?: string;
+  overflow?: string;
+  flex?: GrowShrinkOptions;
```

Comment:
don't think we should expose this property. Too flex specific.

Reply:
Its inorder for us to avoid declaring the flexGrow="1" and shrink="1"
Most of the places need this

Reply:
I was thinking we cold add a grow and shrink prop

Reply:
Or maybe just one of them and by default we always grow?